akka-javasdk-annotation-processor-tests/mcp-endpoints-descriptors/src/main/java/com/example/BasicMcpEndpoint.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example;
+
+import akka.javasdk.annotations.mcp.McpEndpoint;
+import akka.javasdk.annotations.mcp.McpTool;
+
+@McpEndpoint(
+    serverName = "my-mcp-server",
+    serverVersion = "1.0",
```

Comment:
Will there be support to pick, the servierVersion in this case, from a system property or an environment variable though?

Reply:
Something like
````
    serverVersion = ${SERVER_VERSION},
```